.github/workflows/docker-image.yml

```diff
@@ -11,16 +13,72 @@ env:
   IMAGE_NAME: ${{ github.repository }}
 
 jobs:
-  build-and-push-image:
+  build-and-upload-image-to-release:
```

Comment:
Did you try this job anywhere? As far as I understand it will modify the release to add the Docker image. Could you confirm?

Reply:
Yes